CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## To Be Released
+* feat(prober): change the exposed CheckOneProbe to make it usable outside of this package
```

Comment:
picky picky picky
```suggestion

* feat(prober): change the exposed CheckOneProbe to make it usable outside of this package
```

Some Markdown parsers make it mandatory to have a new line after a title.

Reply:
> Missing spec for the method `CheckOneProbe`?

Specs added